### PR TITLE
doc: mimic rc1 release notes

### PR DESCRIPTION
**Pull request by theanalyst**
Adding a starter release notes for mimic, including the PendingReleaseNotes
entries and a few from the trello merged section. This should evolve over the
time when we release RCs where we need to x-ref the features with the docs and
checklist that all the features are indeed in the changes section.

### CODE REVIEW

**Comment by liewegas** [comment]
Looks good.  I think we're mostly ready to do 13.0.2; everything but rgw is clean but those failures are challenging.

File: doc/releases/mimic.rst [hunk 1]
```diff
@@ -0,0 +1,2301 @@
...
+~~~~~~~~~~~~~~~
+
+* *core*:
+
+  - The `pg force-recovery` command will not work for erasure-coded PGs when a
+    Luminous monitor is running along with a Mimic OSD. Please use the
+    recommended upgrade order of monitors before OSDs to avoid this issue.
+
+  - It is no longer possible to adjust ``pg_num`` on a pool that is still being
+    created.
```
Comment (liewegas) [inline doc/releases/mimic.rst]: This one should be dropped; we didn't end up merging this change for mimic.

**Comment by theanalyst** [comment]
rebased and dropped the change

**Comment by theanalyst** [comment]
jenkins test this please

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by theanalyst** [comment]
jenkins test this please

**Pull request merged**